client/a8c-for-agencies/sections/overview/body/hosting/index.tsx

```diff
@@ -1,5 +1,91 @@
+import page from '@automattic/calypso-router';
+import { useDispatch } from '@wordpress/data';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback } from 'react';
+import Offering from 'calypso/a8c-for-agencies/components/offering';
+import PressableLogo from 'calypso/a8c-for-agencies/components/pressable-logo';
+import WordPressLogo from 'calypso/components/wordpress-logo';
+import { recordTracksEvent } from 'calypso/state/analytics/actions/record';
+import './styles.scss';
+
+const A4A_HOSTING_MARKETPLACE_LINK = '/marketplace/hosting';
+
 const OverviewBodyHosting = () => {
-	return <div>Hosting</div>;
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const actionHandlerCallback = useCallback(
+		( section: string, product: string ) => {
+			dispatch(
+				recordTracksEvent( 'calypso_a4a_overview_click_open_marketplace', {
+					section,
+					product,
+				} )
+			);
+		},
+		[ dispatch ]
+	);
+
+	const pressable = {
+		title: 'Pressable',
+		titleIcon: <PressableLogo size={ 24 } />,
+		description: translate(
+			'Pressable offers world-class managed WordPress hosting for agencies with award-winning support, powerful site management, and flexible plans that scale with your business.'
+		),
+		highlights: [
+			translate( 'Git integration, WP-CLI, SSH, and staging.' ),
+			translate( 'Lightning-fast performance.' ),
+			translate( '100% uptime SLA.' ),
+			translate( 'Smart, managed plugin updates.' ),
+			translate( 'Comprehensive WP security with Jetpack.' ),
+			translate( '24/7 support from WordPress experts.' ),
+		],
+		// translators: Button navigating to A4A Marketplace
+		buttonTitle: translate( 'Explore Pressable' ),
+		expanded: true,
+		actionHandler: () => {
+			actionHandlerCallback( 'hosting', 'pressable' );
+			page( A4A_HOSTING_MARKETPLACE_LINK );
```

Comment:
I think since we have a dedicated page for Pressable, we can redirect to `/marketplace/hosting/pressable` and we can keep this constant in https://github.com/Automattic/wp-calypso/blob/093abd50576b2b29536eac84fbf1a6746744f351/client/a8c-for-agencies/components/sidebar-menu/lib/constants.ts

Reply:
Hmm, I'm wondering if we should move all links then outside from the components, and have a `constants` folder inside `a8c-for-agencies`, so we show they are shared?

Reply:
Also, until I think until that PR merged these links wouldn't work. What do you say if we update these later, when marketplace page is fully functional?

Reply:
I think it's ok as we haven't released this product. We could also add a `// FIXME` comment so we don't miss it.
I will leave this up to you to decide 🙂